app/components/BookItem/index.tsx

```diff
@@ -154,16 +159,40 @@ function BookItem(props: Props) {
         {
             refetchQueries: CART_ITEMS_NAME ? [CART_ITEMS_NAME] : undefined,
             onCompleted: (response) => {
-                if (!response?.createCartItem?.ok) {
+                if (response?.createCartItem?.ok) {
+                    alert.show(
+                        strings.bookOrderSuccessMessage,
+                        { variant: 'success' },
+                    );
+                } else if (response?.createCartItem?.errors) {
+                    const transformedError = transformToFormError(
+                        removeNull(response?.createCartItem?.errors) as ObjectError[],
+                    );
                     alert.show(
-                        strings.bookOrderFailedMessage,
+                        <div>
+                            <div>
+                                {strings.bookOrderFailedMessage}
+                            </div>
+                            {isDefined(transformedError) && (
+                                <div>
+                                    {transformedError[internal]}
+                                </div>
+                            )}
+                        </div>,
```

Comment:
@subinasr , lets create a separate component for this. I'll merge this for today, but do this tomorrow plz.